Hunting Queries/SigninLogs/AADSuspectedBruteForce.yaml

```diff
@@ -0,0 +1,38 @@
+id: 056ceb9b-8f07-42b3-853e-ef3779de222e
+name: Suspected Brute force attack Investigation
+description: |
+  'Summarize all the failures and success events for all users in the last 24 hours, 
+  only identify users with more than 100 failures'
+requiredDataConnectors:
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - SigninLogs
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - AADNonInteractiveUserSignInLogs
+tactics:
+  - CredentialAccess
+relevantTechniques:
+  - T1110
+query: |
+    let authenticationWindow = 24h;
+    let successCodes = dynamic(["0", "50125", "50140", "70043", "70044"]);
+    let aadFunc = (tableName:string){
+      table(tableName)
+      | where TimeGenerated > ago(authenticationWindow)
+      | extend Activities = pack("datetime", TimeGenerated,"ResultEventId", ResultType , "AppDisplayName", AppDisplayName, "ResultDescription", ResultDescription ,"IpAddress", IPAddress, "DeviceDetail", todynamic(DeviceDetail), "Status", todynamic(DeviceDetail), "Locationdetails", todynamic(LocationDetails))
```

Comment:
Doing this pack is highly intensive on a workspace with a large amount of data.  We should summarize with these fields directly and then pack what you want after the summarize.
I would not pack IPAddress so this can be used as an Entity.

Reply:
@shainw Please check #3601

Reply:
Removed.